opengovsg/askgovsg
# test(web): add unit tests

## Problem

Web module has no tests to ensure that it is working as expected in the future.

## Solution

Add tests for web module.

**Features**:

- Tests for web service.
- Tests for web controller.

Note: Tests for web routes have _not_ been implemented.

## Tests

1. Click around to make sure all pages are loaded as usual.
2. Change URL to test if page redirects to not-found if agency or post does not exist.
3. Test URLs on Facebook Sharing Debugger to ensure that meta tags are still picked up.
